webrtc-stats.html

```diff
@@ -3622,6 +3632,107 @@ <h2>
           </section>
         </div>
       </section>
+      <section id="stunserverconnection-dict*">
+        <h3>
+          <dfn>RTCStunServerConnectionStats</dfn> dictionary
+        </h3>
+        <div>
+          <pre class="idl">dictionary RTCStunServerConnectionStats : RTCStats {
+             DOMString url;
+             DOMString address;
+             long port;
+             DOMString protocol;
+             RTCNetworkType networkType;
+             unsigned long totalRequestsSent;
+             unsigned long totalResponsesReceived;
```

Comment:
I don't know much about STUN/TURN so feel free other people to pitch in, this looks sensible to me though.

Is there something worth adding for indicating whether or not this is still "active", such as the timestamp of the last request/response or some info if we lose connection, etc?